daemon/src/oracle.rs

```diff
@@ -186,11 +185,12 @@ impl Actor {
                         .await
                         .context("Failed to deserialize as Attestation")?;
 
-                    this.send(NewAttestationFetched {
-                        id: event_id,
-                        attestation: Attestation(attestation),
-                    })
-                    .await?;
+                    let _ = this
```

Comment:
Why are we dropping this error here?

Reply:
woops, must of been because the function now returns an error